lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -2436,6 +2436,25 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :external => :infinite_tracing,
           :description => "Configures the TCP/IP port for the Trace Observer Host"
+        },
+        :'infinite_tracing.compression_level' => {
+          :default => :none,
+          :public => true,
```

Comment:
After the discussion this morning, would it be best if these were left as non public for now? Once we plan to actually turn it on, we could flip it to be public. It's not clear to me if this is planned on being completed for our next release.

Reply:
It'd be great to get clear guidance from Erik on this one. Good catch, Tanna.

Reply:
Good call. Set to `false` with 873ed0a48aabb6c4137d69c2bc5a20457733553c